applications/endoscopy_tool_tracking/python/package-app.sh

```diff
@@ -32,18 +33,15 @@ if [ $(get_host_arch) == "aarch64" ]; then
     PLATFORM=igx-orin-devkit
 fi
 
-echo -e "Copying the required files to the application directory..."
-cp "$GIT_ROOT/install/lib/*" "$APP_PATH"
-cp "$GIT_ROOT/install/lib/gxf_extensions/*" "$APP_PATH"
-cp -R "$GIT_ROOT/install/python/lib/holohub" "$APP_PATH"
+echo -e "Updating application configuration file..."
 sed -i 's|lib/gxf_extensions/||' "$APP_PATH/endoscopy_tool_tracking.yaml"
 echo -e "done\n"
 
 echo -e Install Holoscan CLI and then use the following commands to package and run the Endoscopy Tool Tracking application:
 echo -e "Package the application:"
-echo -e "${YELLOW}holoscan package -c $APP_PATH/endoscopy_tool_tracking.yaml --platform [igx-orin-devkit | jetson-agx-orin-devkit | sbsa, x64-workstation] --platform-config [igpu | dgpu] -t holohub-endoscopy-tool-tracking-python $APP_PATH/endoscopy_tool_tracking.py --include onnx holoviz${NOCOLOR}"
+echo -e "${YELLOW}holoscan package -c $APP_PATH/endoscopy_tool_tracking.yaml --platform [igx-orin-devkit | jetson-agx-orin-devkit | sbsa, x64-workstation] --platform-config [igpu | dgpu] -t holohub-endoscopy-tool-tracking-python --include onnx holoviz --add $INSTALL_DIR/lib/ --add $INSTALL_DIR/python/lib/ $APP_PATH/endoscopy_tool_tracking.py${NOCOLOR}"
 echo -e "\nFor example:"
-echo -e "${YELLOW}holoscan package -c $APP_PATH/endoscopy_tool_tracking.yaml --platform ${PLATFORM} --platform-config ${GPU} -t holohub-endoscopy-tool-tracking-python $APP_PATH/endoscopy_tool_tracking.py --include onnx holoviz${NOCOLOR}"
+echo -e "${YELLOW}holoscan package -c $APP_PATH/endoscopy_tool_tracking.yaml --platform ${PLATFORM} --platform-config ${GPU} -t holohub-endoscopy-tool-tracking-python --include onnx holoviz --add $INSTALL_DIR/lib/ --add $INSTALL_DIR/python/lib/ $APP_PATH/endoscopy_tool_tracking.py${NOCOLOR}"
```

Comment:
Is `--add` recursive, or do we also need `--add $INSTALL_DIR/lib/gxf_extensions/` to match `cp` commands above?

Reply:
It is recursive.